docs/docs/graphql-client-plugin.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: docs
+title: GraphQL client plugin - Merkur
+---
+
+# GraphQL client plugin
+
+The GraphQL client plugin adds `graphql` property to your widget with a `request` method. Under the hood this plugin uses [HTTP Client plugin](https://merkur.js.org/docs/http-client-plugin)
+
+## Installation
+
```

Comment:
Missing part: User must add graphql loader to webpack config.

Reply:
Added